.github/workflows/gh-pages.yml

```
@@ -0,0 +1,36 @@
+name: Publish Storybook
+on:
+  push:
+    branches:
+      - develop
+      - EGRC-275
+
+jobs:
+  publish:
+    name: Publish Storybook to GitHub Pages
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Setup NodeJS
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14
+          cache: "npm"
+      - name: Globally update npm
+        run: npm install -g npm@latest
+      - name: Install dependencies
+        run:
+          npm install
+      - name: Build Storybook
+        run:
+          npm run build-storybook
+      - name: Deploy to Pages
+        uses: JamesIves/github-pages-deploy-action@4.1.4 
+        with:
+          branch: storybook-docs
+          folder: storybook-static
+          clean: true
```

```suggestion
          clean: true
          git-config-name: Easy Dynamics Automation
          git-config-email: noreply@easydynamics.com
```

I think we can workshop these names; however, without something like this, each commit author will be set to the value in the `github` context in GitHub Actions, which will likely be the person who pushed to `develop` (so in our case, the person who merged the PR). That's _fine_ but it may not really be what we want. So we may want to fake a "service account" type person for this. If we let it use the merger's name/email we may have some commits marked as "Unverified" since some folks on the project use PGP keys; also the person merging may not be the person who wrote the change which may slightly obscure authorship info.

I am not 100% confident in this suggestion; there's quite a few pros and cons either way so maybe don't accept this suggestion until we have some sort of conversation around it?

---

Thanks for the feedback! 
I think this is a good change; it wouldn't make sense to have the merger be the author.

---

Ya know, since we've gone ahead and set the `single-commit: true` flag, let's go ahead and roll with this; if we decide to change our mind later (to use a different name/email), it's not we'll have an invalid history we've got to clean up (since we won't have a history at all). And we have one more chance to review this in #113